dosbox.txt

```diff
@@ -8,6 +8,7 @@ URLSite = http://www.dosbox.com/
 URLDownload = http://download.sourceforge.net/project/dosbox/dosbox/0.74-2/DOSBox0.74-2-win32-installer.exe
 SHA1 = 06f23be4ceba35c8c547a98f1b327da20082b483
 SizeBytes = 1453151
+Icon = dosbox.ico
```

Comment:
There is no need for this if the icon name is exactly the same as the txt filename.
The reason we have this field is so that we can share one icon for multiple versions (f.e. multiple versions of chromium using the same icon)

Reply:
I will solve it in diablo2 and dosbox too. :D